src/stylus/coins.styl

```diff
@@ -0,0 +1,54 @@
+// Styles for coins awarded at end of challenges and cases
+w = 256px;
+h = 256px;
+
+bronze = 0;
+silver = 1;
+gold = 2;
+
+single = 1;
+whole = 0;
+
+verticalOffset(awardLevel)
+  h * awardLevel * -1
+
+horizontalOffset(pieces, piece, pieceStyle)
+  // because css background offsets are negative from rhs for reasons
+  totalWidth = (w * pieces * 2) + w
+  rightOffset = (w * pieces * pieceStyle)
+  pieceOffset = totalWidth - rightOffset - (w*piece)
+  pieceOffset * -1
+
+coinPieces = {
+  two: 2,
+  three: 3,
+  four: 4,
+  five: 5
+}
+
+.coin
+  position: absolute;
+  height: h;
+  width: w;
+  background-image: url("../resources/images/coin-background.png");
+  &.piece
+    for pieces in 2..5
+      // coins in 2 3 4 or 5 pieces
+      for pieceNum in 1..pieces
+        // for each piece in pieces
+        &.pieces{pieces}
```

Comment:
Would it make sense to add

`background-image: url("../resources/images/coin-pieces-(pieces).png");`

right here, and then you wouldn't have to also add the background image-name calculation to challenge-award.js? It seems like it would be nice to have it all defined in the CSS, instead of partly in the CSS and partly in the JS (especially as it hard-codes this specific image set in that component).